ansible/roles/test/tasks/interface.yml

```diff
@@ -56,3 +56,6 @@
 - name: Verify VLAN interfaces are up correctly
   assert: { that: "'{{ ansible_interface_facts[item]['active'] }}' == 'True'" }
   with_items: "{{ minigraph_vlans.keys() }}"
+
+- name: Verify interfaces counters
+  include: interface_counters.yml
```

Comment:
I wonder if we should this new test as pytest?

Reply:
"I wonder if we should this new test as pytest?" - in my opinion we should add this in pytest and other features which are executed during run ansible test cases.

Here in ansible approach, this specific verification is a part of sanity check which performs before and after each test case. Currently in pytest approach there is no such functionality.

I think first we should understand and agree:
what steps should be done before and after each test case.
If current ansible logic is enough and its not redundant then it can be ported to the pytest.

Then this verification and more others will be executed per test case.

Reply:
@lguohan is this closed and we can merge?

Reply:
@lguohan I agree with @yvolynets-mlnx here, I think we should add this check and port it back to pytest with the rest of the interface tests.